src/main/java/com/buddy/api/domains/account/entities/AccountEntity.java

```diff
@@ -0,0 +1,59 @@
+package com.buddy.api.domains.account.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Table(name = "person")
+public class AccountEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @EqualsAndHashCode.Include
+    @Column(name = "account_id", nullable = false, unique = true)
+    private UUID personId;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+
+    @Column(name = "email", nullable = false, unique = true)
+    private String email;
+
+    @Column(name = "phone_number")
+    private String phoneNumber;
+
+    @Column(name = "password", nullable = false)
+    private String password;
+
+    @Column(name = "terms_of_user_consent", nullable = false)
+    private Boolean termsOfUserConsent;
+
+    @Column(name = "creation_date", updatable = false)
```

Comment:
```suggestion
    @Column(name = "creation_date", nullable = false, updatable = false)
```

mesmo que o dado já vai ser adicionado automaticamente, ainda acho interessante explicitamente colocar que `creation_date` é `NOT NULL`, para a constraint existir também a nível de banco e deixar claro que essa regra existe.